student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, age, classroom)
+    super(name, age)
+    @classroom = classroom
+  end
```

Comment:
- [ ] [Optional] It would be nice to have a getter method for the `classroom` attribute. This will make it possible for you to access the `classroom` attribute. It will be something like this:

`attr_reader :classroom`.